pywbem/tupleparse.py

```diff
@@ -421,7 +426,7 @@ def parse_declaration(self, tup_tree):
 
         self.check_node(tup_tree, 'DECLARATION')
 
-        child = self.one_child(tup_tree, ['DECLGROUP'])
+        child = self.one_child(tup_tree, ('DECLGROUP'))
```

Comment:
For a single item, the `( ... )` syntax does not create a tuple, but ignores the parenthesis:

```
$ python -c "t=('NAME'); print(type(t))"
<type 'str'>
```

I do not understand why the tests pass, because the use of the `acceptable` parameter in `one_child()` should then test against the one string that comes in. Probably they pass because we don't have tests that try an element name that is just the begin of a valid element name.

The solution is to use a comma after the one item, e.g.:

```
$ python -c "t=('NAME',); print(type(t))"
<type 'tuple'>
```

Reply:
Agreed and you fixed